### DISCUSSION TITLE
What is the proccess for pushing updates to the repo?

### DISCUSSION THREAD

**Discussion by kodecreer** [hsahovic/poke-env#373] [Q&A]
Hi, so I have been making some modifications to make it work with Generation 9 for my project and wondering what is the process you would like  for me to take to push them?

**Comment by hsahovic** [comment]
Hi @kodecreer ! Apologies for not answering earlier.
I would recommend opening an issue, with potentially an accompanying PR. If you're unsure about the best way to solve a problem / whether this should be part of poke-env, issues would be a natural place to do so. Open a PR once you are certain that your solution would fit well and passes all tests.
